src/FixerConfiguration/AliasedFixerOptionBuilder.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\FixerConfiguration;
+
+/**
+ * @author ntzm
+ *
+ * @internal
+ */
+final class AliasedFixerOptionBuilder
```

Comment:
If we decide to go for this decorator approach, would be worth making a `FixerOptionBuilderInterface` that this and `FixerOptionBuilder` implement

Reply:
I'd rather add a `setAlias()` method to `FixerOptionBuilder`.

Reply:
I'd rather add a `setAlias()` method to `FixerOptionBuilder`.

Reply:
@julienfalque see: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3568#discussion_r170391280